src/html/sandbox.html

```diff
@@ -65,27 +65,66 @@
             {{/each}}
         </script>
         <script>
-            var source = document.getElementById('edit_template').innerHTML;
-            var htmlTemplate = Handlebars.compile(source);
-
             Handlebars.registerHelper('boldIfDefault', function (template) {
                 if (!template['issuetype-field'] && !template['projects-field']) {
                     return 'font-bold';
                 } else {
                     return '';
                 }
             })
-
             window.addEventListener('message', function (event) {
                 var command = event.data.command;
-                if (command === "render") {
-                    event.source.postMessage({
-                        html: htmlTemplate(event.data.context),
-                        callback: event.data.callback,
-                        openTemplate: event.data.openTemplate
-                    }, event.origin);
+                switch(command) {
+                    case "renderTemplates":
+                        var source = document.getElementById('edit_template').innerHTML;
+                        var htmlTemplate = Handlebars.compile(source);
+                        event.source.postMessage({
+                            content: 'template-editor',
+                            html: htmlTemplate(event.data.context),
+                            callback: event.data.callback,
+                            openTemplate: event.data.openTemplate
+                        }, event.origin);
+                        break;
+                    case "renderDomain":
+                        var domainSource = document.getElementById('addDomains').innerHTML;
+                        var htmlDomainTemplate = Handlebars.compile(domainSource);
+                            event.source.postMessage({
+                            content: 'domain-list',
+                            html: htmlDomainTemplate(event.data.context),
+                            callback: event.data.callback
+                        }, event.origin);
+                        break;
                 }
             });
         </script>
+
+        <script id="addDomains" type="text/x-handlebars-template">
```

Comment:
I think this should be moved above the section above.

Reply:
:+1: